HashMapFile.java

```diff
@@ -0,0 +1,42 @@
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+
+    public class HashMapFile {
+        public static void main(String[] args) throws IOException {
+
+            File file = new File("src/test/java/file");
+            Map<String, String> allFileMap = new HashMap<>();
+
+            FileReader fr = new FileReader(file);
+            BufferedReader br = new BufferedReader(fr);
+
+            String line;
+            int key = 1;
+
+            while ((line = br.readLine()) != null) {
+                allFileMap.put(String.valueOf(key), line);
+                key++;
+                System.out.println(line);
+            }
+            br.close();
+            fr.close();
+
+            File file1 = new File("src/test/java/powerOfTwo.txt");

```

Comment:
try to avoid names like **file1, variable2, list3** etc.
use something like **fileToWrite**
names should be meaningful

Reply:
fixed a545608